Query Insights 2.19 documentation (#9120)
opensearch-project/documentation-website: _observing-your-data/query-insights/grouping-top-n-queries.md

The response includes the following fields.

Field | Data type        | Description
:--- |:-----------------| :---
`top_queries` | Array            | The list of top query groups.
`top_queries.timestamp` | Integer          | The execution timestamp for the first query in the query group.
`top_queries.id` | String           | The unique identifier for this query or query group.
`top_queries.phase_latency_map` | Object           | The phase latency map for the first query in the query group. The map includes the amount of time, in milliseconds, that the query spent in the `expand`, `query`, and `fetch` phases.
`top_queries.source` | Object           | The first query in the query group.

> `top_queries.group_by` | String           | The group_by setting when this query was executed.

Reviewer: ```suggestion
`top_queries.group_by` | String           | The `group_by` setting applied when the query was executed.
```

> `top_queries.group_by` | String           | The `group_by` setting applied when the query was executed.

Reviewer: I think "was" makes more sense because it's in the past.

`top_queries.total_shards` | Integer          | The number of shards on which the first query was executed.

> `top_queries.node_id` | String           | The node ID of the coordinator node of the first query in the query group.

Reviewer: ```suggestion
`top_queries.node_id` | String           | The node ID of the node that coordinated the execution of the first query in the query group.
```

> `top_queries.search_type` | String           | The search request execution type (`query_then_fetch` or `dfs_query_then_fetch`). For more information, see the `search_type` parameter in the [Search API documentation]({{site.url}}{{site.baseurl}}/api-reference/search/#query-parameters).

Reviewer: Unfortunately, no, because the link is to a table, and they need to look up a specific parameter in that table.

> `top_queries.indices` | Array            | The indexes searched by the first query in the query group.

Reviewer: ```suggestion
`top_queries.indices` | Array            | The indexes to which the first query in the query group is applied.
```

`top_queries.task_resource_usages` | Array of objects | The resource usage breakdown for the various tasks belonging to the first query in the query group.

> `top_queries.query_hashcode` | String           | The hash code that uniquely identifies the query group, generated from the [query structure](#grouping-queries-by-similarity).

Reviewer: ```suggestion
`top_queries.query_hashcode` | String           | The hash code that uniquely identifies the query group and is generated from the [query structure](#grouping-queries-by-similarity).
```

> `top_queries.query_hashcode` | String           | The hash code that uniquely identifies the query group. This is essentially the hash of the [query structure](#grouping-queries-by-similarity).

Reviewer: ```suggestion
`top_queries.query_hashcode` | String           | The hash code that uniquely identifies the query group, generated from the [query structure](#grouping-queries-by-similarity).
```
